README.md

```diff
@@ -324,33 +324,39 @@ Explicit:
 Unchecked
 ```
 
-Once your Google account is registered and the music is uploaded, extract the cookies for our test suite via:
+Once your Google account is registered and the music is uploaded, set the following environment variables
 
 ```
-# Enter into the node REPL
-node
-
-# Inside of the REPL, dump our cookies
-var browser = require('wd').remote();
-browser.init({browserName: 'chrome'}, console.log);
-// Wait for browser window to open
-browser.get('https://play.google.com/music/listen', console.log);
-// Wait for redirect to accounts.google.com
-// Manually log in to page
-// When you are logged in to Google Music, dump the output of the following into `test/cookies.json`
-browser.allCookies(function (err, cookies) { fs.writeFileSync('test/cookies.json', JSON.stringify(cookies, null, 4), 'utf8'); });
+GOOGLE_MUSIC_JS_USER = {your_google_account_username}
```

Comment:
Please rename this to "email" so we can be consistent with Google's terminology

![selection_202](https://cloud.githubusercontent.com/assets/902488/11830048/1cb8b0c4-a367-11e5-811a-ed3a3b5ee8ef.png)